AntistasiOfficial.Altis/resourcecheck.sqf

```diff
@@ -152,15 +152,15 @@ while {true} do {
 		};
 	} forEach ciudades;
 
-	if(count mrkSupplyCrates < 6) then
+	if(countSupplyCrates < 6) then
```

Comment:
I know is WiP, just noticing that countSupplyCrates is still an array so it would require to be
    if (count countSupplyCrates < 6) then